dotnet/src/Connectors/Connectors.Memory.CosmosDB/Connectors.Memory.CosmosDB.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <RepoRoot>$([System.IO.Path]::GetDirectoryName($([MSBuild]::GetPathOfFileAbove('.gitignore', '$(MSBuildThisFileDirectory)'))))</RepoRoot>
+  </PropertyGroup>
+  <Import Project="$(RepoRoot)/dotnet/nuget/nuget-package.props" />
+  
+  <PropertyGroup>
```

Comment:
I expect more details here like AssemblyName, RootNamespace and nuget details (PackageId, Title).

Reply:
Looks like these got lost in the MemoryStore refactor. I'll re-add them back and disable nuget publishing for this project for now.